### ISSUE TITLE
Various Issues with UI interactions 

### ISSUE THREAD

**Issue by Krumelur** [xibbon/XogotIssues#6]
My game is using a resolution of 320x200 pixels. Everything is tiny and I usually  work at a 200% to 400% scale.
On the iPad I am facing multiple  issues:

1. Scrolling the editor area requires 2 fingers which also triggers a zoom sometimes
2. When searching for an asset, the input field's font is black and hardly readable (see video)
3. The search does not expand folders. If the asset I am looking for is in a subfolder, it won't be visible until I expand the folder.
4. I need to zoom into tilemaps (I said, the pixels are tiny on modern screen like the iPad's :-)). This scrolls the tilemap out of view. While there is a button to get it back into view, it will center on the tilemap. The sensitivity of scrolling (which requires two fingers) is way too high. I tend to scroll past the area im interested in (see video)
5. When I want to place a tile, I sometimes need to adjust the area in the editor. Using two fingers to scroll and/or zoom will then place two tiles, or, sometimes place tiles at what seems to be random positions (see video)
6. When closing a scene, a system dialog pops up asking whether I want to leave without saving and it is hard to follow. It asks if I want to save before closing and offers as options: "Don't save", "OK", and "Cancel"; The option to not save and lose all work should not be first. "OK" requires thinking: "OK, don't save?" or "OK, close and save?". In addition, the dialog is half German/half English. I'd prefer to set the locale for Xogot to English although my iPad is set to German.

<img width="1180" alt="image" src="https://github.com/user-attachments/assets/c54f89d0-152c-4ee4-9470-dfe7e70a0040">

Video is currently being processed: https://www.youtube.com/watch?v=1LnJgZcPljM

**Comment by migueldeicaza** [comment]
Internal Bugs tracking:

- ~File Pad search color: #681~
- ~FilePad search unfolding: #682~
- ~Scrolling sensitive: #683~
- ~Spurious events: #684~
- ~Usability: #685~

**Comment by migueldeicaza** [comment]
Fixed
